src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -331,7 +332,11 @@ def _merge(self, dict_1, dict_2, identifier):
                 merged_dict[key] = list(merged_dict[key] + [value])
             elif isinstance(value, list):
                 cleaned_list = list(filter(None, chain(value, merged_dict[key])))
-                merged_dict[key] = list(set(cleaned_list))
+                if len(cleaned_list) > 0 and isinstance(cleaned_list[0], dict):  # make dicts hashable
+                    cleaned_list = [freeze(d) for d in cleaned_list]
```

Comment:
While it is harmless in this context at this point, reassigning a variable with a certain data structure to a different data structure could be prone to bugs. I would not recommend implementing this pattern.